docker-build.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# define paths
+SCRIPT=$(readlink -f "$0")
+DIR=$(dirname "$SCRIPT")
+BUILD_COMMON="$DIR/common"
+BUILD_FBG="$DIR/fbg-server"
+EXPORT_FILE="freeboardgames.tar"
+
+# ensure yarn and npm are installed
+yarn -v > /dev/null 2>&1 || (echo "ERROR: yarn is not installed." && exit 1)
+npm -v > /dev/null 2>&1  || (echo "ERROR: npm is not installed." && exit 1)
+
+# define functions
+confirm() {
+    # call with a prompt string or use a default
+    read -r -p "$@"" [y/N]: " response
+    case "$response" in
+        [yY][eE][sS]|[yY])
+            return 0
+            ;;
+        *)
+            return 1
+            ;;
+    esac
+}
+
+install_fbg() {
+    cd "$BUILD_FBG"
+    read -p "Now fbg will be installed and started. Shut down the server after startup with CTRL+C. [PROCEED]"
+    yarn install && yarn start || exit 1
+}
+
+run_gameserver() {
+    cd "$DIR"
+    read -p "Now the webserver and backend will start. Shut down the server after startup with CTRL+C. [PROCEED]"
+    yarn run dev || (echo -e "ERROR. (ensure node is up-to-date)" && exit 1)
```

Comment:
You can replace this with `yarn run gen:games` instead, so user won't need to issue a Ctrl-C

Reply:
Nice - was looking for this one.